modules/shared/src/main/scala/org/tessellation/cutoff/LogarithmicOrdinalCutoff.scala

```diff
@@ -0,0 +1,37 @@
+package org.tessellation.cutoff
+
+import org.tessellation.schema.SnapshotOrdinal
+
+object LogarithmicOrdinalCutoff {
```

Comment:
I would rename it to something like `LogarithmicOrdinalDistribution` with a `calculate` method. Based on this desired distribution we then remove undesired files. Not a priority at all.